cmd/dep/godep_importer_test.go

```diff
@@ -395,3 +338,13 @@ func equalImports(a, b []godepPackage) bool {
 
 	return true
 }
+
+// projectRootPtr takes a ProjectRoot value and returns a pointer.
+func projectRootPtr(p gps.ProjectRoot) *gps.ProjectRoot {
```

Comment:
I can do that, if you'd like me to. The reason I have it this way is to be able to represent the difference between e.g. an empty `notWantedProjectRoot` (`""`) and a test case without a project root (`nil`). Otherwise we could not test for the case where `notWantedProjectRoot` is empty.

Let me know which approach you'd like and I'll adjust accordingly.